### PR TITLE
Update schedule

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: data/schedule.json [hunk 1]
```diff
@@ -119,7 +129,7 @@
       {
         "startTime": "09:00",
         "endTime": "13:30",
-        "title": "Morning Sessions"
+        "title": "🏽‍🏫 Core Courses Part II & Elective Courses"
```
Comment (daviddias) [inline data/schedule.json]:
```suggestion
        "title": "👩🏽‍🏫 Core Courses Part II & Elective Courses"
```

**Comment by daviddias** [comment]
@cwaring is it ok to merge this? Anything else you would like to see done first?

**Comment by cwaring** [comment]
This is an improvement for now so I'll merge this down, thanks. The data was originally sourced from https://docs.google.com/spreadsheets/d/1oN9_9HaGYcdVEpEAiGwJo5VgekiEkCYA9GbS1FQzbH0/edit?ts=5cda806b#gid=1355058122 so it might be worth updating that too.

I will be changing the format of the schedule in the next release as I begin to migrate all the session data to markdown.

**Comment by daviddias** [comment]
@cwaring good call! Just updated it :)

**Pull request merged**